packages/azure-http-specs/specs/azure/client-generator-core/usage/main.tsp

```diff
@@ -90,11 +91,31 @@ namespace ModelInOperation {
   op modelInReadOnlyProperty(@body body: RoundTripModel): {
     @body body: RoundTripModel;
   };
+
+  @doc("""
+    Serialize the 'OrphanModel' as request body.
+    
+    Expected body parameter: 
+    ```json
+    {
+      "name": "name",
+      "desc": "desc"
+    }
+    ```
+    """)
+  @global.Azure.ClientGenerator.Core.convenientAPI(false)
+  @route("/orphanModelSerializable")
+  @put
+  op orphanModelSerializable(@body body: unknown): NoContentResponse;
```

Comment:
I used the `unknown` to avoid ref the `OrphanModel`.

Let me know any language that having difficulty to serialize a `OrphanModel` model into this op.